easybuild/easyblocks/p/psmpi.py

```diff
@@ -92,8 +93,14 @@ def configure_step(self):
             self.cfg.update('configopts', ' --with-mpichconf="%s"' % self.cfg['mpich_opts'])
 
         # Lastly, set pscom related variables
-        self.cfg.update('preconfigopts', ('PSCOM_LDFLAGS=-L{0}/lib '+
-                'PSCOM_CPPFLAGS=-I{0}/include').format(get_software_root('pscom')))
+        if self.cfg['pscom_allin_path'] is None:
+            self.cfg.update('preconfigopts', ('PSCOM_LDFLAGS=-L{0}/lib '+
+                    'PSCOM_CPPFLAGS=-I{0}/include').format(get_software_root('pscom')))
+        else:
+            pscom_allin_path = self.cfg['pscom_allin_path'].strip()
```

Comment:
Why do you need the `.strip()` here, that's a bit strange?

Reply:
Otherwise there was an empty space that messed up the definition of the variable